src/app.jsx

```diff
@@ -794,30 +784,42 @@ class Application extends React.Component {
             </AlertGroup>
         );
 
+        const contextInfo = {
+            cgroupVersion: this.state.cgroupVersion,
+            registries: this.state.registries,
+            selinuxAvailable: this.state.selinuxAvailable,
+            podmanRestartAvailable: this.state.podmanRestartAvailable,
+            userPodmanRestartAvailable: this.state.userPodmanRestartAvailable,
+            userLingeringEnabled: this.state.userLingeringEnabled,
+            version: this.state.version,
+        };
+
         return (
-            <WithDialogs>
-                <Page id="overview" key="overview">
-                    {notificationList}
-                    <PageSection className="content-filter" padding={{ default: 'noPadding' }}
-                                 variant={PageSectionVariants.light}>
-                        <ContainerHeader
-                            handleFilterChanged={this.onFilterChanged}
-                            handleOwnerChanged={(_event, value) => this.onOwnerChanged(value)}
-                            ownerFilter={this.state.ownerFilter}
-                            textFilter={this.state.textFilter}
-                            twoOwners={this.state.systemServiceAvailable && this.state.userServiceAvailable}
-                            user={this.state.currentUser}
-                        />
-                    </PageSection>
-                    <PageSection className='ct-pagesection-mobile'>
-                        <Stack hasGutter>
-                            { this.state.showStartService ? startService : null }
-                            {imageList}
-                            {containerList}
-                        </Stack>
-                    </PageSection>
-                </Page>
-            </WithDialogs>
+            <WithPodmanInfo value={contextInfo}>
+                <WithDialogs>
+                    <Page id="overview" key="overview">
+                        {notificationList}
+                        <PageSection className="content-filter" padding={{ default: 'noPadding' }}
+                                     variant={PageSectionVariants.light}>
+                            <ContainerHeader
+                                handleFilterChanged={this.onFilterChanged}
+                                handleOwnerChanged={(_event, value) => this.onOwnerChanged(value)}
+                                ownerFilter={this.state.ownerFilter}
+                                textFilter={this.state.textFilter}
+                                twoOwners={this.state.systemServiceAvailable && this.state.userServiceAvailable}
+                                user={this.state.currentUser}
+                            />
+                        </PageSection>
+                        <PageSection className='ct-pagesection-mobile'>
+                            <Stack hasGutter>
+                                { this.state.showStartService ? startService : null }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1249-20230622-093407-daa4f85f-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)